### DISCUSSION TITLE
[com_finder] Is there a way to automatically exclude common words from searches?

### DISCUSSION THREAD

**Discussion by Chaosxmk** [joomla/joomla-cms#40159] [Q&A] [answered]
I've built a site with a metric ton of pages, so the smart search system is very helpful for finding the specific documents users need. However, french has some filler words I'd like to exclude from searches (ex: à, au, on, etc).

Is there a relatively quick and painless way to handle this?

**Comment by brianteeman** [comment, marked as answer]
yes there is a way. Look in the english language folder and you will see a commonwords file

**Reply by Chaosxmk** [reply]
So if I add words to the file, they will automatically be excluded? Or is there something else I'd need to do?

**Reply by brianteeman** [reply]
also make sure you enable it in the component options
![image](https://user-images.githubusercontent.com/1296369/226440886-cf18bdb5-e95b-42fb-9d39-1a9185fe8ee0.png)

**Comment by Hackwar** [comment]
The file would only work when the language is installed. To add additional words manually, you can add them in the #__finder_terms_common table. Since this solves this discussion, I'm closing it.
